.pre-commit-config.yaml

```diff
@@ -1,33 +1,77 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+minimum_pre_commit_version: '2.8.2'
+
+default_language_version:
+  python: python3
+
+default_stages: [commit]
```

Comment:
Why are these new defaults needed?

Reply:
The `default_language_version` to ensure Python 3 is used on hooks that don't specify it, but I _think_ its redundant nowadays? Doesn't hurt to have it, but _probably_ doesn't hurt to remove it either?

The `default_stages` ensures that `pre-commit run --hook-stage manual` only runs the `manual` hooks (i,e. `codespell`, and any others we may add); otherwise, it would run all hooks and there wouldn't be a way to run _only_ manual hooks, which often comes in handy.